src/psyclone/psyGen.py

```diff
@@ -6224,6 +6357,15 @@ def view(self, indent=0):
     def __str__(self):
         return "Reference[name:'" + self._reference + "']"
 
+    def math_equal(self, other):
+        '''Returns true if the self has the same results as other.
+        :param other: the node to compare self with.
+        :type other: py:class:`psyclone.psyGen.Node`
```

Comment:
Please document the :returns: and :rtype:

Reply:
Done